### PR TITLE
Setup for manylinux1 scripts

### PR DESCRIPTION
**Pull request by jetuk**
This was based on https://github.com/pypa/python-manylinux-demo and seems to be the way to do binary wheel releases for Linux.

### CODE REVIEW

**Comment by jetuk** [comment]
This should work fine with the recipe changes in #10

File: build-wheels.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+./configure
+make
+make check
+make install
+
+
+cd ${WORKDIR}
+
+# Fetch Pywr
+# Old version of wget/OpenSSL in the manylinux image does not with Github.
```
Comment (snorfalorpagus) [inline build-wheels.sh]: "does not work"

File: README.md [hunk 2]
```diff
@@ -11,3 +18,35 @@ Release process:
...
+```
+
+After completing the wheels should be located in the `wheelhouse` folder of the `pywr-deploy`
+repository. These can then be uploaded to pypi using `twine`.
+
+```
+pipenv run twine upload wheelhouse/*
+```
+ 
+Note there is no deployment of Python 2.x wheels. It's too much effort.
```
Comment (snorfalorpagus) [inline README.md:52]: This is OK.

**Comment by jetuk** [comment]
I've removed the Pipefile for twine.

**Pull request merged**